README.md

```diff
@@ -145,16 +145,36 @@ There is only a limited number of such operations and they are only implemented
+* **Start with a miniscule frontend improvement** like a [less than 5 line UI change](https://github.com/theia-ide/theia-trace-extension/pull/369/files). The Theia-based frontend repo is much less complex than the Trace Server's, so by starting in the frontend you'll learn high-level context about the project's different components without being overwhelmed with complexity. 
+  * [Good first issues](https://github.com/theia-ide/theia-trace-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+  * _Develop with Gitpod to initially avoid dev setup._ Put `gitpod.io/#` before your project fork's github URL to open an in-browser IDE and initially avoid any local dev setup. This dev approach works best for small changes because Gitpod workspaces are volatile. 
+  * _Learn [how to contribute code](https://github.com/theia-ide/theia-trace-extension#how-to-contribute-code)._ Starting with a tiny change means you can first focus on exploring the code as well as learning the submission and review process (which is a significant learning step if you've never submitted a PR before).
+
 ## How to contribute code
-**Changes to the project** are made by submitting code with a pull request (PR).
+**Changes to the project** are made by submitting code with a pull request (PR). 
 
 * [How to write and submit changes](https://www.dataschool.io/how-to-contribute-on-github/)
+* [Example pull request](https://github.com/theia-ide/theia-trace-extension/pull/402)
 
 **Good commit messages** make it easier to review code and understand why the changes were made. Please include a:
 * _Title:_ Concise and complete title written in imperative (e.g. "Update Gitpod demo screenshots" or "Single-click to select or open trace")
 * _Problem:_ What is the situation that needs to be resolved? Why does the problem need fixing? Link to related issues (e.g. "Fixes [#317](https://github.com/theia-ide/theia-trace-extension/issues/317)").
 * _Solution:_ What changes were made to resolve the situation? Why are these changes the right fix?
 * _Impact:_ What impact do these changes have? (e.g. Numbers to show a performance improvement, screenshots or a video for a UI change)
 * [_Sign-off:_](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) Use your full name and a long-term email address. This certifies that you have written the code and that, from a licensing perspective, the code is appropriate for use in open-source.   
-* [How to format the message](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
 
+Other commit information:
+* [How to format the message](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+* [Example commit message](https://github.com/ebugden/theia-trace-extension/commit/3c19ae7950a749f2a66b4665ec9f7ef3c907089d) (TODO Replace with persistent commit after [related-repos PR](https://github.com/theia-ide/theia-trace-extension/pull/402) merge)
```

Comment:
Need to replace the commit message link with the persistent commit (after merging [related-repos PR](https://github.com/theia-ide/theia-trace-extension/pull/402))